src/tls/Makefile

```diff
@@ -368,7 +368,7 @@ mitls.cmxa: record_fstar_version $(LIBFILES)
 	$(OCAML) -a $(OCAML_INCLUDE_PATHS) $(TLSML:.ml=.cmx) -I $(OUTPUT_DIR) -o mitls.cmxa
 
 #16-10-19 beware: failing tests in mitls.cmxa will leave mitls.exe unchanged.
-mitls.exe:  mitls.cmxa 
+mitls.exe:  test/mitls.ml mitls.cmxa
```

Comment:
Agreed, but I would rather spend my energy switching to scons than cleaning up this makefile.  The missing dependency on test/mitls.ml broke me while I was writing new code in it.